lib/ios-crash-log.js

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+import { asyncmap } from 'asyncbox';
+
+const glob = B.promisify(_glob);
+
+
+const logger = getLogger('IOS-crash-log');
```

Comment:
think we should call this logger 'iOSCrashLog', also no need for extra spaces above it